cli/flox/src/config/mod.rs

```diff
@@ -70,6 +70,9 @@ pub struct FloxConfig {
     #[serde(default)]
     pub trusted_environments: HashMap<EnvironmentRef, EnvironmentTrust>,
 
+    /// The url of the floxhub instance to use
+    pub floxhub_url: Option<Url>,
```

Comment:
nit: a bit weird to call this floxhub_url, when it could be some other domain. E.g. if this was containers I would have `registry: hub.docker.com | ghcr.io`, not `dockerhub: hub.docker.com | ghcr.io`. For now it's just floxhub though so no need to waste time on it

Reply:
Well I just don't know _what_ our floxhub is. I think we never named it.
I do agree though in principle 👍🏼